hack/release.toml

```diff
@@ -19,117 +18,6 @@ preface = """\
     [notes.updates]
         title = "Component Updates"
         description="""\
```

Comment:
the flannel update should be here right?

Reply:
I have two PRs, I can't do both at once :-D